pyt/utils/log.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
This file is completely unused but while developing you can do

```python
from pyt.utils.log import enable_logger, logger
enable_logger(to_file='./pyt.log')
```

to log to a file.